packages/cf-deploy-config-inquirer/test/validators.test.ts

```diff
@@ -230,7 +230,12 @@ describe('validateMtaId long Windows path', () => {
         };
         const combinedLength = `${longPath}\\${input}`.length;
         const result = validateMtaId(input, previousAnswers);
-        expect(result).toBe(t('error.windowsMtaIdPathTooLong', { length: combinedLength }));
+        expect(result).toBe(
+            t(
```

Comment:
I couldn't understand how this was working, as you are still using the `t` for translating but add a full string. 

I checked out the branch and you are mocking the `t` function, everything is returning `MTA ID too long` so the tests are not correct